go.mod

```
@@ -10,29 +10,34 @@ require (
 	github.com/gorilla/mux v1.8.0
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
 	github.com/spf13/cast v1.5.0
-	github.com/spf13/cobra v1.4.0
+	github.com/spf13/cobra v1.5.0
 	github.com/stretchr/testify v1.8.0
-	github.com/tendermint/tendermint v0.35.4
+	github.com/tendermint/tendermint v0.35.2
```

[question] this line may not be used b/c the `replace` later in this file but out of curiosity, why v0.35.2 here instead of v0.34.20

```suggestion
	github.com/tendermint/tendermint v0.34.20
```

---

btw removing that line doesn't seem to affect the project. Did it locally and was able to run everything correctly.

---

yeah, its still working since we are blindly forcing all uses of tendermint to celestia-core instead of only instance of v0.34.20

good catch tho